com.sublimehq.SublimeText.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: com.sublimehq.SublimeText
+command: subl
+
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.gnome.SettingsDaemon
+  - --filesystem=host
+  - --env=PATH=/app/utils/bin:/app/sublime_merge/bin:/app/bin:/usr/bin
+  - --env=PYTHONPATH=/app/utils/lib/python3.8/site-packages
+
+add-extensions:
+  com.sublimemerge.App:
+    directory: sublime_merge
+    add-ld-path: lib
+
+modules:
+  - name: sublime
+    buildsystem: simple
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - FLATPAK_ID=com.sublimehq.SublimeText
+          # Extract upstream package
+          - mkdir -p deb-package export/share
+          - ar p sublime.deb data.tar.xz | tar -xJf - -C deb-package
+          # Install desktop entry and icons
+          - mv deb-package/usr/share/{applications,icons} export/share/
+          - rename sublime_text ${FLATPAK_ID} export/share/applications/*.desktop
+          - rename sublime-text ${FLATPAK_ID} export/share/icons/hicolor/*/apps/*
+          - desktop-file-edit --set-key=Exec --set-value='subl %F' --set-key=Icon
+            --set-value=${FLATPAK_ID} export/share/applications/${FLATPAK_ID}.desktop
+          # Install the app
+          - mv deb-package/opt/sublime_text ./
+          - ln -sr /app/sublime_merge/extra/sublime_merge ./sublime_merge
+          # Cleanup
+          - rm -rf deb-package sublime.deb
+      - type: script
+        dest-filename: subl
+        commands:
+          - exec /app/extra/sublime_text/sublime_text --fwdargv0 "$0" "$@"
+      - type: extra-data
```

Comment:
Please add configuration for https://github.com/flathub/flatpak-external-data-checker/ to make sure it gets automatically updated. When it's in, it's good to be merged.